api-references/provider-rest-api.md

```diff
@@ -528,18 +528,91 @@ Response:
 ```json
 [
     {
-        "cpuType":"AMD Ryzen 7 5800X 8-Core Processor"
-        "currentJobs":0
-        "desc":"This is a mocked enviroment"
-        "diskGB":2
-        "gpuType":"AMD RX570"
-        "id":"ocean-compute"
-        "maxJobs":10
-        "nCPU":2
-        "nGPU":0
-        "priceMin":2.3
+        "cpuType":"AMD Ryzen 7 5800X 8-Core Processor",
+        "currentJobs":0,
+        "desc":"This is a mocked enviroment",
+        "diskGB":2,
+        "gpuType":"AMD RX570",
+        "id":"ocean-compute",
+        "maxJobs":10,
+        "nCPU":2,
+        "nGPU":0,
+        "priceMin":2.3,
         "ramGB":1
     },
     ...
 ]
 ```
+
+### Authentication endpoints
+
+Provider offers an alternative to signing each request, by allowing users to generate auth tokens.
+The generated auth token can be used until its expiration in all supported requests.
+Simply omit the signature parameter and add the AuthToken request header based on a created token.
+
+Please note that if a signature parameter exists, it will take precedence over the AuthToken headers.
+All routes that support a signature parameter support the replacement, with the exception of auth-related ones
+(createAuthToken and deleteAuthToken need to be signed).
+
+#### GET /api/services/createAuthToken
+
+Allows the user to create an auth token.
+
+Parameters
+
+```
+address: String object containing consumer's address (optional)
+nonce: Integer, Nonce (required)
+signature: String object containg user signature (signed message)
+ The signature is based on hashing the following parameters:
+   address + nonce
+expiration: valid future UTC timestamp (required)
+```
+
+Returns:
+Created auth token.
+
+Example:
+
+```
+GET /api/services/createAuthToken?address=0xA78deb2Fa79463945C247991075E2a0e98Ba7A09&&nonce=1644317370&&expiration=1660053210&signature=0x70895648fb9957537b0ffd0d12a9705dda35445e5e5b6b40b7b7448902a29e194773b10d8fee1750d9eb5542e252610d4aa7edfc3101fb8fa9d2c45729f0698301
```

Comment:
The tricky thing about providing examples like this is that it implies that if you send this request you'll get the example response. When sending this request, the actual response is `Invalid timestamp provided.`. I would suggest changing the values so that it is more obvious that they are placeholders e.g. `address=0x123ABC` and adding a note below saying that the placeholder values values need to be replaced with actual values. Same also applies for the DELETE request